orte/mca/plm/slurm/plm_slurm_module.c

```diff
@@ -422,10 +422,10 @@ static void launch_daemons(int fd, short args, void *cbdata)
 
     if (0 < opal_output_get_verbosity(orte_plm_base_framework.framework_output)) {
         param = opal_argv_join(argv, ' ');
-        OPAL_OUTPUT_VERBOSE((1, orte_plm_base_framework.framework_output,
-                             "%s plm:slurm: final top-level argv:\n\t%s",
-                             ORTE_NAME_PRINT(ORTE_PROC_MY_NAME),
-                             (NULL == param) ? "NULL" : param));
+        opal_output(orte_plm_base_framework.framework_output,
```

Comment:
I would say change this to `opal_output_versbose` (lower case) so it is on when verbose is enabled. But since the containing if statement already checks that then this is fine.

Reply:
We talked with @jjhursey. `opal_output` seems ok in this scenario.
I'm merging it.